Formula/ejabberd.rb

```diff
@@ -20,7 +21,7 @@ class Ejabberd < Formula
   depends_on "erlang"
   depends_on "gd"
   depends_on "libyaml"
-  depends_on "openssl@1.1"
+  depends_on "openssl@3"
```

Comment:
For most formulae, we probably want to keep `openssl@1.1` until other changes are in place.

See:
- https://github.com/Homebrew/homebrew-core/issues/104784#issuecomment-1174378680
- https://github.com/Homebrew/homebrew-core/pull/85775#issuecomment-934666042

Reply:
just keep using `openssl@1.1`